src/storage/write_ahead_log/disk_log_consumer_task.cpp

```diff
@@ -42,8 +42,7 @@ void DiskLogConsumerTask::WriteBuffersToLogFile() {
 }
 
 uint64_t DiskLogConsumerTask::PersistLogFile() {
-  // buffers_ may be empty but we have callbacks to invoke due to read-only txns
```

Comment:
Note: this comment wasn't true and misunderstood what `buffers_` contains. It can't ever be empty if logging is enabled.